homeassistant/components/peco/strings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "PECO Outage Counter",
+        "description": "Please choose your county below.",
```

Comment:
This feels kinda duplicate/assumed. The field below already says that; removing this line will not change anything, so I suggest removing it.